bundles/org.openhab.binding.senechome/src/main/java/org/openhab/binding/senechome/internal/SenecHomeApi.java

```diff
@@ -71,17 +69,26 @@ public void setHostname(String hostname) {
      */
     public SenecHomeResponse getStatistics()
             throws InterruptedException, TimeoutException, ExecutionException, IOException {
-        String location = HTTP_PROTO_PREFIX + hostname;
+        String location = hostname + "/lala.cgi";
+        logger.trace("sending request to: {}", location);
+
+        try {
+            httpClient.start();
+        } catch (Exception e) {
+            logger.error("cannot connect", e);
+        }
```

Comment:
The HTTP client is created by the handler factory. Please start it in the handler factory.
**And also stop it.**

Reply:
You have an example here:
https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.linky/src/main/java/org/openhab/binding/linky/internal/LinkyHandlerFactory.java#L89